src/Files.App/DataModels/NavigationControlItems/DriveItem.cs

```diff
@@ -221,17 +223,17 @@ public int CompareTo(INavigationControlItem other)
 
 		public async Task LoadDriveIcon()
 		{
-			if (IconData is null)
+            if (IconData is not null)
 			{
-				if (!string.IsNullOrEmpty(DeviceID))
-					IconData = await FileThumbnailHelper.LoadIconWithoutOverlayAsync(DeviceID, 24);
-
-				if (IconData is null)
-				{
-					var resource = UIHelpers.GetIconResourceInfo(Constants.ImageRes.Folder);
-					IconData = resource?.IconData;
-				}
-			}
+                Icon = await IconData.ToBitmapAsync();
+				return;
+            }
+         
```

Comment:
The formatting here is spacing instead of tabs

Reply:
Should I change the `IconData is not null` and invert it to not repeat the `Icon = await IconData.ToBitmapAsync()`